packages/auth0-api-js/EXAMPLES.md

```diff
@@ -0,0 +1,25 @@
+# Examples
+
+- [Get an access token for a connection](#get-an-access-token-for-a-connection)
+
+## Get an access token for a connection
```

Comment:
There should be more detailed docs around each of the parameters. Use the [existing docs](https://github.com/guabu/auth0-auth-js/blob/main/packages/auth0-auth-js/EXAMPLES.md#retrieving-a-token-for-a-connection) as a reference.
<img width="1105" height="493" alt="image" src="https://github.com/user-attachments/assets/cc078135-2b52-4cb1-a903-37a2bf1ccaa6" />

Reply:
Also, we are not describing here the shape of the `tokenSet` variable (ie. the return type of the method). I'd recommend that we somehow show its shape, at the very least, that the AT is available at `tokenSet.accessToken`.

Reply:
non blocking ^